### DISCUSSION TITLE
Continuation of repo

### DISCUSSION THREAD

**Discussion by JeffreyArts** [strapi-community/plugin-io#98] [General]
Hi @ComfortablyCoding,

I just installed this Strapi plugin and I miss a feature in it to extend a request like I can do with the Strapi REST flow via a controller. I notice that the demo / documentation site is old (it works, but I don't expect the example page to be updated anytime soon for instance 😉), and it lacks certain features that I need. 

This made me wonder what your position is in the development upon this plugin. Is this something you'd like to continue working on? Or is it no longer maintained?

**Comment by ComfortablyCoding** [comment]
Hello!

This plugin is no longer being maintained by me for some time now. It was moved to the community repository with the hopes that someone else would pick up stewardship.

**Reply by JeffreyArts** [reply]
Okay, can you elaborate how that'd work? Since that sounds to me that no single person is responsible for the development / roadmap of this plug-in. Making it impossible to process PR's or handling bugfixes/features, stalling the progress of this plug-in.

**Reply by ComfortablyCoding** [reply]
Anyone interested in maintaining the repo would contact @derrickmehaffy requesting ownership. Once they get access they are free to update/maintain as needed. Happy to discuss if needed with any initial help getting up and running/handoff with whoever picks it up.

**Reply by derrickmehaffy** [reply]
Indeed if anyone would like to take up ownership of this you can either reach out to me on the community discord (https://discord.strapi.io) or you can reach out to me by email: derrick.mehaffy@strapi.io

Thanks!
